examples/file-uploader/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "file-uploader-example",
+  "version": "0.0.1",
+  "description": "File uploader example",
+  "main": "index.js",
+  "scripts": {
+    "pretest": "browserify js/test.js --debug -t /usr/lib/node_modules/babelify --outfile js/test_build.js",
+    "test": "node js/test_build.js | ./node_modules/tap-spec/bin/cmd.js",
+    "build": "browserify js/main.js --debug -t /usr/lib/node_modules/babelify --outfile js/build.js"
```

Comment:
Yeah... this is a hack to get it to work on my box that I accidentally left in. Will fix. (Essentially -- windows file system is a bit of a challenge for npm -- no symlinks and no deep paths. For related reasons babelify is not listed as a dep -- will fix that too).